README.md

```diff
@@ -89,6 +90,9 @@ It currently supports several remote state backend providers:
 - [Terraform Cloud (remote)](https://www.terraform.io/docs/backends/types/remote.html)
 - [GitLab](https://docs.gitlab.com/ee/user/infrastructure/terraform_state.html)
 
+With the upcoming **v1.2.0** update, Terraboard will be now able to handle multiple buckets/providers configuration! 🥳
```

Comment:
That's a nice addition. We'll just need to remember to remove it when we release 😄

Reply:
```suggestion
With the upcoming **v2.0.0** update, Terraboard will be now able to handle multiple buckets/providers configuration! 🥳
```